CMakeLists.txt

```diff
@@ -154,6 +155,12 @@ list(APPEND COMPILER_SRCS ${RELAY_BACKEND_SRCS})
 list(APPEND COMPILER_SRCS ${RELAY_IR_SRCS})
 list(APPEND COMPILER_SRCS ${RELAY_QNN_SRCS})
 
+if(USE_OMP)
+  message(STATUS "Build with OpenMP")
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fopenmp")
```

Comment:
Jon, I updated the cmakefile. It'd work for Windows. However, I don't have a Windows dev machine. Could you help me verify whether it works on Windows?

Reply:
Checking now.

Reply:
On Windows, the file is `libiomp5md.lib`, from the docs [here](https://scc.ustc.edu.cn/zlsc/sugon/intel/compiler_f/main_for/optaps/common/optaps_par_libs.htm).

Can you please update the Intel block to look like this:
`
if(MSVC)
    find_library(OMP_LIBRARY NAMES libiomp5md)
else
    find_library(OMP_LIBRARY NAMES iomp5)
`

Reply:
Thanks, Jon!
Yes, we can potentially add an option for openmp path just like MKL. Do you it's necessary?

Reply:
It's probably not necessary for now, we can always add it in a later change :) also, just to make sure, when using OpenMP, TVM_NUM_THREADS will not be used, right? We will only use OMP_NUM_THREADS?

Reply:
You can use either TVM_NUM_THREADS or OMP_NUM_THREADS, but OMP_NUM_THREADS is recommended since it is effective to 3rd party library using OMP as well.
You can see the number of threads is determined at https://github.com/dmlc/tvm/pull/4089/files#diff-f67700f4f293bfeb6dbc32bd659f4e48R405
https://github.com/dmlc/tvm/blob/master/src/runtime/threading_backend.cc#L218